spa/public/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Do we need this html file? If we run a `yarn build`, doesn't that create the html file?

Reply:
Yes we need this, as the reactscripts relies on this.